src/components/VCard.vue

```diff
@@ -1,11 +1,33 @@
 <template>
   <div class="card">
-    <img
-      v-if="image"
-      class="card-image"
-      :src="image"
-      alt="alt"
+    <a
+      v-if="titleLink"
+      class="card-image-link"
+      :href="titleLink"
+      aria-hidden="true"
+      role="presentation"
+      tabindex="-1"
     >
+      <img
+        v-if="image"
+        class="card-image"
+        :src="image"
+        :alt="title || alt"
+        role="presentation"
```

Comment:
don't think you need the role here as well, the parent one applies to the child as well

Reply:
Wasn't sure about this one, but looked into it and it's apparently only inherited in certain cases like rows of tables or items of lists but doesn't affect the semantics of children otherwise. https://www.w3.org/TR/wai-aria-1.1/#presentation 

We're in a weird case here because we need the link tag to have accessible text, so the image needs alt text, but also we actually want it all to be hidden as it's redundant with the following link.